packages/cli/src/types.ts

```diff
@@ -88,3 +88,29 @@ export interface deploymentDataType {
     };
   };
 }
+
+export interface projectDataType {
```

Comment:
Types, Interfaces and Classes should start with a capital

```suggestion
export interface ProjectDataType {
```